### ISSUE TITLE
[EPR] Resolve whether EPR manifest is like a WebApp Manifest or a /.well-known 

### ISSUE THREAD

**Issue by hillbrad** [w3c/webappsec#373]
WebApp Manifests may be loaded cross-origin (with credentials!) and there may be multiple manifests at a given origin, each with their own scope, representing multiple installable applications.  EPR refers to discovering and loading a manifest by reference to the [MANIFEST] algorithm, but then seems to indicate that such a manifest is one-per-origin and must only be loaded same origin.

We need to decide whether an EPR manifest is like (or potentially an attribute of) a WebApp Manifest, or whether it is like a /.well-known resource and behave consistently.  Monkey-patching the WebApp Manifest algorithm is probably not the best way forward.

**Comment by randomdross** [comment]
Yes, this is an important issue to resolve.  The way I had been thinking about it:
  - The way is now (EPR enabled/disabled for the whole origin) is intentional and simple.  It's problematic for large domains though and I had always considered that at some point we'd switch to a more flexible model.
  - I had initially thought that the EPR: 1 could be augmented with a path to a manifest.  Though that's not compatible with the WebApp Manifest spec.
  - Conceptually, consistency with the WebApp Manifest spec has always seemed like a great goal.  But the actual details of this could get hairy (eg: "with credentials!")

So now I'm thinking maybe we do go the rfc5785 /.well-known route.  We could register /.well-known/epr and have the default manifest live at /.well-known/epr/manifest.  Then if we wanted to allow a given path tree /foo/bar to have its own manifest then content under that path would serve a "EPR:1; localPath" header, enabling EPR for everything under /foo/bar, with a manifest of /.well-known/epr/foo/bar/manifest.

Thoughts?
